spdx-utils/src/test/kotlin/ExtensionsTest.kt

```diff
@@ -40,4 +43,18 @@ class ExtensionsTest : WordSpec({
             sum shouldBe enumSetOf(DayOfWeek.MONDAY, DayOfWeek.TUESDAY, DayOfWeek.WEDNESDAY)
         }
     }
+
+    "URI.stripUserInfo()" should {
+        "remove only the user info given a URI with username" {
+            val uri = URI("ssh://user@abc.com/some/path.git?some=query#withfragment").stripUserInfo()
```

Comment:
Better use example.com instead of abc.com, because abc.com belongs to a commercial company while example.com exists to be used in examples.

Reply:
Interresting, done.